src/app/services/login-service/login-service.service.ts

```diff
@@ -0,0 +1,28 @@
+import { Injectable } from '@angular/core';
+import { HttpClient, HttpHeaders, HttpParams } from '@angular/common/http';
+import { Observable } from 'rxjs';
+import { environment } from 'src/environments/environment';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class LoginService {
+
+  constructor(private http: HttpClient) { }
+  login(username, password): Observable<any> {
+    const headers = new HttpHeaders({
+      'Content-Type': 'application/json',
+      'Access-Control-Allow-Origin': '*'
+    });
+    const options = {
+      headers: headers,
+      observe: 'response' as 'body'
+    };
+
+    return this.http.post(`${environment.loginUrl}`,
+      {
+        'username': `${username}`,
+        'password': `${password}`,
+      }, options);
+  }
+}
```

Comment:
it is awesome! good job táska! you should show me this and implement it into the register form